pandas/util/testing.py

```diff
@@ -992,11 +992,18 @@ def raise_assert_detail(obj, message, left, right, diff=None):
         left = pprint_thing(left)
     elif is_categorical_dtype(left):
         left = repr(left)
+
```

Comment:
can you import PY2 and string_types up top

Reply:
👍575b2e8